adapters/telaria/telaria.go

```diff
@@ -205,6 +205,9 @@ func (a *TelariaAdapter) MakeRequests(requestIn *openrtb2.BidRequest, reqInfo *a
 		// Swap the tagID with adCode
 		request.Imp[i].TagID = telariaImpExt.AdCode
 	}
+	if len(request.Imp) > 1 {
```

Comment:
If you are going to drop all but the first imp, do we need to loop over all the imps above?

Also, if there is only 1 imp that goes to the bidder, you know all your bids are for that imp, and don't need to loop over all the imps in `MakeBids()`

Reply:
Thank you for the input. That's a good point; there's no need to loop over all the imps above. I will make changes to that right away.

It's still necessary to loop over all the imps in`makeBids()` because not all responses from the bidder is for the 1 imp that's sent. Our bidder takes in ext information such as pod duration to calculate the number of imps that's most suitable for the pod and return multiple bids for each slot in the pod. Therefore, I need to loop over them and make sure they are for different imp.

Reply:
It looks like you only send the first imp to your bidder. How does your bidder know what the imp IDs for the other imps are in order to bid on them?

Reply:
Similar to the RTB 2.6 spec, our bidder calculates the appropriate pod size based on poddur and maxseq parameters in addition to the configured parameters in the ad unit. Since we cannot really get the poddur information from what's passed to our adapter, we ask it to be present in the video ext. And In the end, we match the bid responses with the imp ids.

Reply:
Do you only support long format pods, or should you verify that the request is such a request before throwing the rest of the imps away? I just want to make sure you can still support the old general video case unless you explicitly are not supporting it. :) Otherwise the PR looks good.

Reply:
Thank you so much for the review and inputs. And this is all we support at the moment :)